packages/calypso-e2e/src/lib/pages/plugins-page.ts

```diff
@@ -19,7 +19,7 @@ const selectors = {
 	browseAllPaid: 'a[href^="/plugins/browse/paid"]',
 	browseFirstCategory: 'button:has-text("Search Engine Optimization")',
 	categoryButton: ( section: string ) => `button:has-text("${ section }")`,
-	breadcrumb: ( section: string ) => `.plugins-browser__header a:text("${ section }") `,
+	breadcrumb: ( section: string ) => `.plugins-browser__header button:text("${ section }") `,
```

Comment:
```suggestion
	breadcrumb: ( section: string ) => `.plugins-browser__header :text("${ section }") `,
```
Does this work? These are already more specific than we want them to be, ideally e2e tests just look for text to click on.